packages/cozy-client/src/models/account.js

```diff
@@ -114,3 +115,23 @@ export const buildAccount = (konnector, authData) => {
     state: null
   }
 }
+
+/**
+ * Look if the given account is connected or not. Does a trigger which is referencing this account
+ * exist
+ *
+ * @param {import('../CozyClient').default} client - CozyClient instance
+ * @param {import('../types').IOCozyAccount} account - account document
+ * @returns {Promise<Boolean>}
+ */
+export const isAccountConnected = async (client, account) => {
```

Comment:
But since the the connected/disconnected state is displayed to the user in harvest, I think this has more meaning than isAccountWithTrigger, no ?

Reply:
The meaning of a function should makes sense to the developer, not the user. And for me it's always better to reflect the actual technical meaning, rather than a "high level" meaning, that will be scope-dependent. Furthermore, we may now use this function out of the scope of harvest ;)

Reply:
Ok, let's fo for isAccountWithTrigger